sfscan.py

```diff
@@ -424,9 +424,9 @@ def __startScan(self):
 
         finally:
             if not failed:
+                self.__setStatus("FINISHED", None, time.time() * 1000)
                 self.runCorrelations()
                 self.__sf.status(f"Scan [{self.__scanId}] completed.")
-                self.__setStatus("FINISHED", None, time.time() * 1000)
```

Comment:
Scans and correlations are two different things. Once a scan is complete we can run correlations as many times as we want. Updating the scan status to `FINISHED` before running correlations makes sense, and lets the Correlator validate that the scan has been completed to prevent attempting to run correlations on running scans.